core/workspace_svg.js

```diff
@@ -715,11 +715,13 @@ Blockly.WorkspaceSvg.prototype.moveDrag = function(e) {
 };
 
 /**
- * Is the user currently dragging a block or scrolling the workspace?
+ * Is the user currently dragging a block or scrolling the flyout/workspace?
  * @return {boolean} True if currently dragging or scrolling.
  */
 Blockly.WorkspaceSvg.prototype.isDragging = function() {
   return Blockly.dragMode_ == Blockly.DRAG_FREE ||
+      (Blockly.Flyout.startFlyout_ &&
```

Comment:
I see.  In that case, I'd prefer that somewhere (maybe just in Blockly.js) you explicitly set startFlyout_ to null and add jsdoc.  Right now it's effectively created in flyout.js and there's no docstring anywhere to explain its intent/why it needs to exist.  In fact, that's why I deleted it last week.